pipelines/multi-branch/tracker.yml

```
@@ -0,0 +1,99 @@
+resource_types:
+- name: git-branches
+  type: registry-image
+  source:
+    repository: aoldershaw/git-branches-resource
+
+- name: terraform
+  type: registry-image
+  source:
+    repository: ljfranklin/terraform-resource
+
+resources:
+- name: feature-branches
+  type: git-branches
+  source:
+    uri: https://github.com/concourse/examples
+    branch_regex: 'feature/(?P<feature>.*)'
+
+- name: examples
+  type: git
+  source:
+    uri: https://github.com/concourse/examples
+
+- name: staging-env
+  type: terraform
+  source:
+    backend_type: gcs
+    backend_config: &terraform_backend_config
+      bucket: concourse-examples
+      prefix: multi-branch/terraform
+      credentials: ((concourse_artifacts_json_key))
```

nit: keep it general, looks like this was copied from one of our pipelines
```suggestion
      credentials: ((artifacts_json_key))
```